pkg/report/writer.go

```diff
@@ -10,16 +10,40 @@ import (
 
 	"golang.org/x/xerrors"
 
-	"github.com/aquasecurity/trivy/pkg/vulnsrc/vulnerability"
+	dbTypes "github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy/pkg/types"
 
 	"github.com/olekukonko/tablewriter"
 )
 
 type Results []Result
 
 type Result struct {
-	FileName        string                                `json:"Target"`
-	Vulnerabilities []vulnerability.DetectedVulnerability `json:"Vulnerabilities"`
+	FileName        string                        `json:"Target"`
+	Vulnerabilities []types.DetectedVulnerability `json:"Vulnerabilities"`
+}
+
+func Write(format string, output io.Writer, results Results, outputTemplate string, light bool) error {
```

Comment:
The name `Write` is confusing in this context where there's an interface `Writer` and a method within that interface also called `Write`. Maybe we should rename this to something else like `WriteOutput` for the lack of a better name?

Reply:
Actually, I didn't feel that this name is good, but I couldn't come up with a good name. I will change Write to WriteOutput. Thanks.

Reply:
I used WriteResults